pkg/target/target.go

```diff
@@ -88,14 +94,68 @@ func (h *K8sValidationTarget) HandleReview(obj interface{}) (bool, interface{},
 		return true, data, nil
 	case *admissionv1beta1.AdmissionRequest:
 		return true, data, nil
-	case SideloadNamespace:
-		return true, &augmentedReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
-	case *SideloadNamespace:
-		return true, &augmentedReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
+	case AugmentedReview:
+		return true, &gkReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
+	case *AugmentedReview:
+		return true, &gkReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
+	case AugmentedUnstructured:
+		admissionRequest, err := augmentedUnstructuredToAdmissionRequest(data)
```

Comment:
`AugumentedUnstructured` should be building a `gkReview` object that has `Unstable.Namespace` defined.

Reply:
Thanks -- done